dev/breeze/src/airflow_breeze/utils/packages.py

```diff
@@ -53,8 +53,8 @@
 
 LONG_PROVIDERS_PREFIX = "apache-airflow-providers-"
 
-# TODO: use single source of truth for those
-# for now we need to keep them in sync with the ones in setup.py
+# TODO(potiuk): use single source of truth for those
```

Comment:
Would this be also a candidate for a separate PR? Instead of putting your name on it, is there an existing issue to point to?

Reply:
This TODO(potiuk) are (were) more about the DRAFT state of the PR - I wasl planning to fix them before I make this PR "ready". And .... It happened.

This comment and TODO are gone now. I simply implemented it. In the new version of this PR, the `devel` dependencies are actually part of the `provider` dependencies generated in pyproject.toml. So this is really part of the same change - we are refactoring and removing dependencies as a "whole" and moving stuff to pyproject.toml for all of the deps (and removing the deps specification from `setup.cfg`).

So rather than moving the devel_deps from `setup.py` to `pyproject.toml` I am moving them to `provider.yaml` now (and pre-commit will add them to the provider dependencies in `pyproject.toml` in the right place, with the right comment).